src/main/kotlin/stringadder/domain/Parser.kt

```diff
@@ -0,0 +1,12 @@
+package stringadder.domain
+
+private val PATTERN = Regex("//(.*)\\\\n(.*)")
```

Comment:
```suggestion
private val PATTERN = Regex("//(.+)\\\\n(.*)")
```
위에가 맞지 않을까 하는데요!! 어떻게 생각하시는가요~?

Reply:
- `+`는 반복을 표현하며 x 문자가 1번 이상 반복됨을 의미
- `*`는 반복여부를 표현하며 x 문자가 0번 이상 반복됨을 의미

그렇네요..! 감사합니다 덕분에 한번 더 숙지하게 됐네요!